### ISSUE TITLE
downRemovalMargin - stack overflow

### ISSUE THREAD

**Issue by nohajc** [TanUkkii007/akka-cluster-custom-downing#19]
Hi, I'm trying to use your library but it seems there's an infinite recursion. I set up the downing strategy according to README but the application crashes immediately with 

```
java.lang.StackOverflowError
	at tanukki.akka.cluster.autodown.OldestAutoDowning.downRemovalMargin(OldestAutoDowning.scala:13)
```

After inspecting the code, it really seems the overridden method `downRemovalMargin` is calling itself.

**Comment by robconrad** [comment]
👍

**Comment by borgmatthew** [comment]
I have submitted a pull request that resolves this issue

**Comment by TanUkkii007** [comment]
fixed in 0.0.12
